pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -83,25 +154,46 @@ func cocoRuntimeAddition(np *v1alpha1.NimbusPolicy, rule v1alpha1.Rule) kyvernov
 							kyvernov1.ResourceFilter{
 								ResourceDescription: kyvernov1.ResourceDescription{
 									Kinds: []string{
-										"apps/v1/Deployment",
+										"v1/ConfigMap",
 									},
 									Selector: &metav1.LabelSelector{
-										MatchLabels: np.Spec.Selector.MatchLabels,
+										MatchLabels: map[string]string{
```

Comment:
Is matchLabels needed ? We can put in the exact name of the config map that will trigger the mutation